examples/systemd/otelcol-sumo.service

```diff
@@ -8,6 +8,7 @@ Group=otelcol-sumo
 MemoryHigh=2000M
 MemoryMax=3000M
 TimeoutStopSec=20
+EnvironmentFile=-/etc/otelcol-sumo/env/*.env
```

Comment:
How does systemd behave if this directory does not exist or is empty?

Reply:
=-  makes the directive optional, so if there is no file to read it... systemd just keeps going.

Reply:
Just confirmed I rename the directory restarted my local otelcol-sumo service..
I'm getting the heartbeat http package in my mock sumologic endpoint but no mysql metrics.
I interpret this to mean.. mysql receiver isn't authing..no password defined... but the other otelcol functionality is working as expected.  systemd isn't trying to do its restaart loop